Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: all test changelog
+
+all: test stmps changelog
+
+VERSION != git describe --tags HEAD
+
+stmps:
+	go build -ldflags="-X main.Version=$(VERSION)" -o stmps .
+
+changelog:
+	git cliff -o CHANGELOG.md
```

Comment:
OMG, keeping `Version` strings up-to-date is a nightmare!

Note it's a bit of a problem in any Go program, because many -- like stmps -- can be `go install`ed, which leaves a useless `Version` string. Better than nothing for the releases and packagers, though. I haven't done this before, but I think we can also do this:

```
if bi, ok := debug.ReadBuildInfo(); ok {
   Version = bi.Main.Version
}
```

which _should_ return the version specified in `go install`. I'll play with it a bit and if it works, I'll submit a PR.

Incidentally, I mentioned elsewhere that I'm maintaining the Alpine package for stmps -- although I don't use Alpine, packaging for it is pretty easy, so I'm doing it. Someone else is maintaining `stmps-git` in Arch, which I *think* is building from `main`. We may want to look into that if `main` is going to have breaking changes -- is a dev branch, in other words. I can talk to the maintainer about fixating the pulled version to specific

Reply:
Wait, that was a brainfart. Editing...

Reply:
It's in AUR as `stmps-git`. That's how I originally found stmps!